frontend/herd.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file This file contains the Herd function. This function fetches herd for a
+ *       given `herdId` (parsed from the url), as well as the individuals
+ *       belonging to that herd.
+ */
+import React from 'react'
+import {Link, useParams} from "react-router-dom";
+
+import { get } from './communication';
+
+/**
+ * Shows herd information, with a list of all individuals belonging to that
+ * herd.
+ */
+export function Herd() {
+  let { herdId } = useParams();
+  const [herd, setHerd] = React.useState(undefined)
+
+  React.useLayoutEffect(() => {
+    get(`/api/herd/${herdId}`).then(
+      data => setHerd(data.herd),
+      error => console.error(error)
+    )
+  }, [])
```

Comment:
This will need `herdId` in the list of dependencies:

```React.useEffect(() => ..., [herdId])```

Also we should use `useEffect` and not `useLayoutEffect`, it was just due to a brainfart I started using the layout version. That one is only useful if you want to synchronously do something with the layouting, such as measuring sizes of dom nodes (like `getClientRect()`)